starlite/app.py

```diff
@@ -512,26 +600,52 @@ def _add_node_to_route_map(self, route: BaseRoute) -> RouteMapNode:
         self._configure_route_map_node(route, current_node)
         return current_node
 
-    def _add_route_to_handler_index(self, route: BaseRoute) -> None:
-        """Maps route handler names to urls.
+    def _get_route_handlers(self, route: BaseRoute) -> List["RouteHandlerType"]:
+        """Retrieve handler(s) as a list for given route."""
+        route_handlers: List["RouteHandlerType"] = []
+        if isinstance(route, (WebSocketRoute, ASGIRoute)):
+            route_handlers.append(route.route_handler)
+        else:
+            route_handlers.extend(cast("HTTPRoute", route).route_handlers)
+
+        return route_handlers
+
+    def _get_handler_qualname(self, handler: "RouteHandlerType") -> str:
+        """Retrieves __qualname__ for passed route handler."""
+        if handler.fn and hasattr(handler.fn, "__qualname__"):
+            return handler.fn.__qualname__
+
+        return type(handler.fn).__qualname__
+
+    def _store_handler_to_route_mapping(self, route: BaseRoute) -> None:
+        """Stores mappings:
+
+            * between route handlers and routes;
+            * between route handlers and names.
```

Comment:
Please remove the bullet points- its incorrect style wise. 

```
Stores the mapping of route handlers to routes and to route handler names. 
```

Alternatively, add a `Notes:` section.